examples/with-msw/pages/index.js

```diff
@@ -1,15 +1,25 @@
 import { useState } from 'react'
 
-export default function Home({ book }) {
+export default function Home({ book, error }) {
   const [reviews, setReviews] = useState(null)
 
   const handleGetReviews = () => {
     // Client-side request are mocked by `mocks/browser.js`.
+    // This request will fail in production, as MSW is only run in development.
+    // In a real-world app this request would hit the actual backend.
```

Comment:
These comments aren't needed because this request won't be executed in production due to the check below.